float64 crosstrackError
float64 headingError
